api/src/pcapi/routes/backoffice/bookings/individual_bookings_blueprint.py

```diff
@@ -404,52 +317,188 @@ def get_batch_cancel_individual_bookings_form() -> utils.BackofficeResponse:
+                        token=token, message=str(exc)
+                    ),
+                    "warning",
+                )
+
+    _flash_success_and_error_messages(success_count, error_dict, False)
+
+
+def _flash_success_and_error_messages(
+    success_count: int, error_dict: dict[str, list[str]], is_validating: bool
+) -> None:
+    if success_count > 0:
+        flash(
+            f"{success_count} {pluralize(success_count, 'réservation a', 'réservations ont')} été {'validée' if is_validating else 'annulée'}{pluralize(success_count)}",
+            "success",
+        )
+
+    if error_dict:
+        if success_count > 0:
+            error_text = Markup("Certaines réservations n'ont pas pu être {action} et ont été ignorées :<br> ").format(
+                action="validées" if is_validating else "annulées"
+            )
+        else:
+            error_text = Markup("Impossible {action} ces réservations : <br>").format(
+                action="de valider" if is_validating else "d'annuler"
+            )
+        for key in error_dict:
+            tokens = error_dict[key]
+            match key:
+                case "BookingIsAlreadyCancelled":
+                    error_text += _build_booking_error_str(
+                        tokens,
+                        f"réservation{pluralize(len(tokens))} déjà annulée{pluralize(len(tokens))}",
+                    )
+                case "BookingIsAlreadyUsed":
+                    error_text += _build_booking_error_str(
+                        tokens,
+                        f"réservation{pluralize(len(tokens))} déjà validée{pluralize(len(tokens))}",
+                    )
+                case "BookingIsAlreadyRefunded":
+                    error_text += _build_booking_error_str(
+                        tokens,
+                        f"réservation{pluralize(len(tokens))} déjà remboursée{pluralize(len(tokens))}",
+                    )
+                case "BookingDepositCreditExpired":
+                    error_text += _build_booking_error_str(
+                        tokens,
+                        f"réservation{pluralize(len(tokens))} dont le crédit associé est expiré",
+                    )
+                case "tooManyBookings":
+                    error_text += _build_booking_error_str(
+                        tokens,
+                        f"réservation{pluralize(len(tokens))} dont l'offre n'a plus assez de stock disponible",
+                    )
+                case "insufficientFunds":
+                    error_text += _build_booking_error_str(
+                        tokens,
+                        f"réservation{pluralize(len(tokens))} dont le crédit associé est insuffisant",
+                    )
+        flash((error_text), "warning")
+
+
+def _build_booking_error_str(tokens: list[str], message: str) -> str:
+    return Markup("- {count} {message} (<a href='{url}'>{tokens}</a>)<br>").format(
```

Comment:
Je pense que ça ferait plus homogène avec le style des liens du BO : 
```suggestion
    return Markup("- {count} {message} (<a class="link-primary" href='{url}'>{tokens}</a>)<br>").format(
```